<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Overview</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget_Projects">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="tableWidget_ProjectInfo">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>0</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_AddProject">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>270</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Project</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionImport_Project"/>
     <addaction name="actionImport_Project_List"/>
    </widget>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionExport_Project"/>
     <addaction name="actionExport_Project_List"/>
    </widget>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionRecent_Projects"/>
    <addaction name="separator"/>
    <addaction name="menuImport"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Project_2"/>
    <addaction name="actionSave_Project_As"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Project"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionRecent_Projects">
   <property name="text">
    <string>Recent Projects</string>
   </property>
  </action>
  <action name="actionSave_Project_As">
   <property name="text">
    <string>Save Project As</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionImport_Project">
   <property name="text">
    <string>Import Project</string>
   </property>
  </action>
  <action name="actionImport_Project_List">
   <property name="text">
    <string>Import Project List</string>
   </property>
  </action>
  <action name="actionExport_Project">
   <property name="text">
    <string>Export Project</string>
   </property>
  </action>
  <action name="actionExport_Project_List">
   <property name="text">
    <string>Export Project List</string>
   </property>
  </action>
  <action name="actionSave_Project_2">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionPreferences_Settings">
   <property name="text">
    <string>Preferences/Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
